alembic/versions/07c6633fa1b6_delete_signature_summaries.py

```diff
@@ -0,0 +1,82 @@
+"""delete signature summaries
+
+Revision ID: 07c6633fa1b6
+Revises: afc2f95a298b
+Create Date: 2017-08-31 17:33:19.969807
+
+"""
+from alembic import op
+from socorro.lib.migrations import load_stored_proc
+
+# revision identifiers, used by Alembic.
+revision = '07c6633fa1b6'
+down_revision = 'afc2f95a298b'
+
+
+def upgrade():
+    # Get a list of ALL tables that start with 'signature_summary_*'
+    connection = op.get_bind()
+    cursor = connection.connection.cursor()
+    cursor.execute("""
+        SELECT c.relname FROM pg_catalog.pg_class c
+        WHERE c.relkind = 'r' AND c.relname LIKE 'signature_summary_%'
+    """)
+    all_table_names = []
+    for records in cursor.fetchall():
+        table_name, = records
+        all_table_names.append(table_name)
+
+    # Now delete all these massive tables
+    for table_name in all_table_names:
+        op.execute('DROP TABLE IF EXISTS {}'.format(table_name))
+
+    # Now delete all stored procedures
+    op.execute("""
+        DROP FUNCTION IF EXISTS
+        update_signature_summary_architecture(date, boolean)
+    """)
+    op.execute("""
+        DROP FUNCTION IF EXISTS
+        update_signature_summary_device(date, boolean)'
```

Comment:
This and the later ones all have a `'` at the end that needs to be removed.

Reply:
I'm pretty sure these get tested if you run `make dockertest`.

Reply:
I ran the migration in bash with alembic in docker. All worked. Then I made the PR, realized I forgot the "IF EXISTS" because reasons reasons. Then when I added that I used a macro to edit multiple lines and screwed it up and noticing that. Then I screwed up correcting my typo.